changelog/next/breaking-changes/3785.md

```diff
@@ -0,0 +1,2 @@
+The "autostart on stop" feature is not available in the pipeline manager
+anymore.
```

Comment:
In this and the other changelog entry, can you refer to the `/pipeline` API instead of the _pipeline manager_, as the former is what the user is interacting with and the latter mostly an implementation detail?

Reply:
Changed in https://github.com/tenzir/tenzir/pull/3785/commits/7d736742695ef5e199b8577633277afa414161ca